packages/preview/codly/1.3.1/README.md

```diff
@@ -0,0 +1,239 @@
+# Codly: simple yet beautiful and powerful code blocks
+
+<p align="center">
+  <a href="https://github.com/Dherse/codly/blob/main/docs.pdf">
+    <img alt="Documentation" src="https://img.shields.io/website?down_message=offline&label=manual&up_color=007aff&up_message=online&url=https%3A%2F%2Fgithub.com%2FDherse%2Fcodly%2Fblob%2Fmain%2Fdocs.pdf" />
+  </a>
+  <a href="https://github.com/Dherse/codly/blob/main/LICENSE">
+    <img alt="MIT License" src="https://img.shields.io/badge/license-MIT-brightgreen">
+  </a>
+  <img src="https://github.com/Dherse/codly/actions/workflows/test.yml/badge.svg" />
+</p>
+
+Codly superchargescode blocks for your Typst documents. It allows you to add annotations, skip lines, customize numberings, add language icons, and much more. Codly is even better with its companion package [`codly-languages`](https://typst.app/universe/package/codly-languages) which provides a wide range of language icons and colors to choose from.
+
+A full set of documentation can be found [in the repo](https://raw.githubusercontent.com/Dherse/codly/main/docs.pdf).
+
+![Example](./assets/demo.png)
+
+````typ
+#import "@preview/codly:1.3.1": *
+#import "@preview/codly-languages:0.1.1": *
```

Comment:
Is there any reason this doesn't use the latest release? Ironically I've seen this in a lot of other packages, and this here might be the reason why :)